.github/workflows/windows.yml

```diff
@@ -1,71 +1,163 @@
 ---
-name: windows-latest
+name: windows
 
 on: [push, pull_request]
+
 env:
   ARCH: "x64"
-  VC: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat"
+  CC: cl
+  CXX: cl
+
+  DOWNWARD_COIN_ROOT_RELEASE: D:\a\downward\osi_release
+  DOWNWARD_COIN_ROOT_DEBUG: D:\a\downward\osi_debug
+  DOWNWARD_CPLEX_ROOT: D:\a\downward\cplex
+  ZLIB_ROOT: D:\a\downward\zlib
+
+  CPLEX_INSTALLER_URL: "${{ secrets.CPLEX129_WINDOWS_URL }}"
+  OSI_URL: "https://www.coin-or.org/download/source/Osi/Osi-0.107.9.tgz"
+  ZLIB_URL: "https://www.zlib.net/zlib1211.zip"
+
+
 jobs:
   compile: # name of the job. Jobs run in parallel unless specified otherwise.
     name: compile
-    runs-on: windows-latest
+    runs-on: ${{ matrix.platform.os }}
     strategy:
       matrix:
-        python-version: ['3.6']
-        build-version: ["release", "debug"]
+        platform:
+          - {os: "windows-2016", vc: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat"}
+          - {os: "windows-2019", vc: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat"}
+        python-version: [3.6]
     steps: # each - is a new sequentially run step
-      - uses: actions/checkout@master
+      - name: clone-repo
+        uses: actions/checkout@master
 
-      - name: Setup python
-        uses: actions/setup-python@v1
+      - name: setup-python
```

Comment:
The step names can include spaces and are usually written as "Set up Python" etc.

Reply:
I would keep this in line with the naming of the other workflows

Reply:
Done.